<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi Chat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background-color: '#0d3d4c';
}</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 700 30pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 1px;
	color: '#e3e3e3';
}</string>
   </property>
   <property name="text">
    <string>Multi Chat</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="createBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 5px solid '#101014';
	font: 700 20pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 1.4px;
	color: '#e3e3e3';
	border-radius: 12px;
}

QPushButton::hover {
	background-color: '#0c2f3a';
}</string>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 5px solid '#101014';
	font: 700 20pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 1.4px;
	color: '#e3e3e3';
	border-radius: 12px;
}

QPushButton::hover {
	background-color: '#0c2f3a';
}</string>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>201</width>
     <height>4</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: '#101014';
	border-radius: 2px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="copyrightLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>364</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 700 8pt &quot;Yu Gothic UI&quot;;
	color: rgb(165, 165, 153);
}</string>
   </property>
   <property name="text">
    <string>Copyright Â© 2022 Vincenzo Caliendo. All rights reserved</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
